### ISSUE TITLE
Would you like the full dataset of Flappy Royale replays to work with? 

### ISSUE THREAD

**Issue by orta** [bryfeng/personal-projects#2]
Hey there @bryfeng - given you've already done a great job with your own hand-made data, we were wondering if you wanted a JSON data dump of _all_ our recorded replays to work from?

There's a strong chance your saves aren't in there ([see here](https://github.com/flappy-royale/flappy-royale/blob/master/notes/server-infrastructure.md)) - but your comparisons could be considerably richer

/cc @lazerwalker

**Comment by bryfeng** [comment]
yes sure, I would love to take a look at the json dump and work with it. Really appreciate the outreach!

**Comment by orta** [comment]
Can you send me an email saying "Hi" - it's on my GitHub profile
